.github/workflows/fixtures.yml

```diff
@@ -40,13 +40,13 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           labels: regenerate-fixtures
           
-      - name: Set up prerequisites (interpreter)
-        if: inputs.tester_repo == 'interpreter-tester'
-        uses: codecrafters-io/codecrafters-action/.github/actions/interpreter-setup@b7d515cea86f1815ce74c6eeb0cb15fb458eda88
-
-      - name: Set up prerequisites (redis)
-        if: inputs.tester_repo == 'redis-tester'
-        uses: codecrafters-io/codecrafters-action/.github/actions/redis-setup@ef5aeb9504e08cccfe71e1dbcf806fab3738019d
```

Comment:
Let's try to store the action in `.github/actions/setup.yml` for all repos, and just invoke that directly here (no conditionals, so we don't need to edit code when new repositories are added)